package.json

```diff
@@ -65,4 +65,4 @@
   "engines": {
     "node": ">=8.0.0"
   }
-}
+}
```

Comment:
minor nit: should we add back the newline at the EOF?

Reply:
Sure, I'll add it back. This got added by the deploy script, so this might happen again next time.